desktop/esrWinFull/distribution/distribution.ini

```diff
@@ -11,12 +11,10 @@ app.distributor="mozillaonline"
 app.distributor.channel="esrWinFull"
 app.partner.mozillaonline="mozillaonline"
 app.chinaedition.channel="esr.firefox.com.cn"
-browser.newtabpage.activity-stream.default.sites=""
+browser.aboutwelcome.enabled=false
 browser.newtabpage.activity-stream.fxaccounts.endpoint="https://accounts.firefox.com.cn"
-browser.newtabpage.activity-stream.topSitesRows=1
-browser.newtabpage.pinned="[{\"url\":\"https://www.baidu.com/index.php?tn=monline_6_dg\",\"label\":\"百度\"},{\"url\":\"https://www.zhihu.com/\",\"label\":\"知乎\",\"customScreenshotURL\":\"https://offlintab.firefoxchina.cn/static/img/as-icons/e78f8151b73defd313f1db76d937157180d8d407.png\"},{\"url\":\"https://www.qq.com/\",\"label\":\"腾讯\",\"customScreenshotURL\":\"https://offlintab.firefoxchina.cn/static/img/as-icons/910f7dca622c0e2455da199624944409a45824aa.png\"},{\"url\":\"https://www.ifeng.com/\",\"label\":\"凤凰网\",\"customScreenshotURL\":\"https://offlintab.firefoxchina.cn/static/img/as-icons/7c2b3ec9646ddef696fba730ee35d7e9ae3ba443.png\"},{\"url\":\"https://weibo.com/\",\"label\":\"微博\",\"customScreenshotURL\":\"https://offlintab.firefoxchina.cn/static/img/as-icons/046255d285ec22213bbe875942377e97f143c576.png\"},{\"url\":\"https://www.taobao.com/\",\"label\":\"淘宝\",\"customScreenshotURL\":\"https://offlintab.firefoxchina.cn/static/img/as-icons/491725fc7b6a0c4dcedf81cfcc52b1a10f961c65.png\"},{\"url\":\"https://www.ctrip.com/\",\"label\":\"携程旅行\",\"customScreenshotURL\":\"https://offlintab.firefoxchina.cn/static/img/as-icons/a4f13a05e39ee6907f7537c5b4c0a52e574e3319.png\"},{\"url\":\"https://www.iqiyi.com/\",\"label\":\"爱奇艺\",\"customScreenshotURL\":\"https://offlintab.firefoxchina.cn/static/img/as-icons/4a112acabb24cadbac7ea4a2027c67fb3e8bcd99.png\"}]"
 browser.showPersonalToolbar=true
-extensions.webservice.discoverURL="https://discovery.addons.mozilla.org/%LOCALE%/firefox/discovery/pane/%VERSION%/%OS%/%COMPATIBILITY_MODE%?edition=china"
+trailhead.firstrun.branches="join-dynamic"
```

Comment:
Mostly to avoid any mention of Monitor, which doesn't work with China FxA.

See https://bugzil.la/1618475 and the same change was made to release channel in #8